src/app/service/photos/photos.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PhotosService {
+
+  constructor(private http: HttpClient) { }
+
+  getPhotos(): Promise<Photo[]> {
+    return this.http.get(`https://${environment.facebookphotosUrl}`, {
+      params: { token: environment.facebookphotosToken }
+    }).toPromise()
+    .then((response: any) =>
+    //is this the correct way to make an api call?
```

Comment:
Good attempt! I noticed you're referring to [this](https://developers.facebook.com/docs/graph-api/reference/photo/) but I think before you can do that, you need to initalise `FB`. Let's resolve this together!